kubernetes/server/create.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
 function kubectl_create() {
-  kubectl create -f scdf.yml
+  kubectl create -f scdf.yml --namespace $KUBERNETES_NAMESPACE
   READY_FOR_TESTS=1
   for i in $( seq 1 "${RETRIES}" ); do
-    SERVER_URI=$(kubectl get svc scdf | grep scdf | awk '{print $3}')
+    SERVER_URI=$(kubectl get svc scdf --namespace $KUBERNETES_NAMESPACE | grep scdf | awk '{print $3}')
     [ '<pending>' != $SERVER_URI ] && READY_FOR_TESTS=0 && break
```

Comment:
@trisberg @viniciusccarvalho What do you think about these changes to better handle netcat_port check: https://github.com/ilayaperumalg/spring-cloud-dataflow-acceptance-tests/commit/65cebd60ba83791580b2e7c2bf6b3dfbed0bbd79

I ran into issues when running the tests against `kubernetes` and with these changes I could run fine.